go.mod

```
@@ -1,5 +1,11 @@
-module github.com/sabhiram/go-gitignore
+module github.com/Diogenesoftoronto/go-gitignore
```

I think we should not change the module name:
```suggestion
module github.com/sabhiram/go-gitignore
```